README.md

```diff
@@ -210,3 +210,54 @@ An example assetlist json contains the following structure:
 }
 ```
 
+## IBC Data
+
+The metadata contained in these files represents a path abstraction between two IBC-connected networks. This information is particularly useful when relaying packets and acknowledgments across chains.
+
+This schema also allows us to provide helpful info to describe open channels.
+
+Note: when created these files, please ensure the the chains in both the file name and `chain-` and `chain-2` are in alphabetical order. Ex: `Achain-Zchain.json`
```

Comment:
minor typo, meant to say 'chain-1' before chain-2, I presume

Reply:
Should we mention that chain-name should match exactly the name of the chain's directory in the chain registry?

Reply:
Fixed and added!